python/setup.py

```diff
@@ -0,0 +1,12 @@
+import setuptools
+
+setuptools.setup(
+    name="hyperspace",
+    version="0.1.0",
+    description="Python bindings for hyperspace.",
```

Comment:
```suggestion
    description="Python bindings for Hyperspace.",
```

Reply:
oh I thought we are putting capital H when it's the first word. I see few places in scala where we use H and h in some places. 
@apoorvedave1  @imback82  @pirz  did we adapt to capital H in general even it is not first word ? Should we create an issue to fix that styling?

Reply:
I believe it is 'H' every where, irrespective of position in a sentence.

Reply:
Thanks @pirz  I think some scala unit tests has 'h'. But I guess we can clean up later. I will change to 'H' then.

Reply:
Updated to  'H' where looked relevant.